Calypso remap and rename

comma opened the pull request:
https://baystation12.net/forums/threads/calypso-remap-rename.5199/

comma committed:
Remaps and renames Calypso to Charon

Also adds handrails to buckle yourself to without a chair.

Crushtoe commented:
👎 Name it Batspider.

ghost commented:
all nobles to charon now

AceVX commented:
Can we not have do a name that could be read as a lifeweb reference? Of course people came out of the woodwork to vote for it because its pandering. 

_"LOL VOTED 4 TEH LIFEWEB 1 xD"_

comma committed:
Made all cameras autoname unless ctag set specifically

Fuuuuuck naming every single camera manually.
Left autoname camera path as mapping stub.
Leaving null ctag test in in case something goes wrong with it in the future.

Kelenius commented:
"I want to make the thing"
"Make a vote first"
"Okay, here's a vote, people want the thing"
"Wait... no... that doesn't count, my opinion is more important"

Can we not.

comma commented:
It's not Lifeweb reference, it's a joke on lethality of away missions.
Get over people picking option you disagree with.
I am fucking done with this.
